templates/smaily-woocommerce-admin.php

```diff
@@ -134,7 +134,7 @@
 						<input
 							id="password"
 							name="password"
-							value="<?php echo ( $result['password'] ) ? $result['password'] : ''; ?>"
+							value="<?php echo ( $result['password'] ) ? esc_html($result['password']) : ''; ?>"
```

Comment:
Formatting: `esc_html` function argument should be wrapped with spaces.

```suggestion
							value="<?php echo ( $result['password'] ) ? esc_html( $result['password'] ) : ''; ?>"
```

Reply:
Added. should install phpcs on my IDE